cookbook/remote_access/inspecting_executions.py

```diff
@@ -2,8 +2,11 @@
 Inspecting Workflow and Task Executions
 ---------------------------------------
 
-Inspecting workflow and task executions are done in the same manner as below. For more details see the
-`flytectl API reference <https://docs.flyte.org/projects/flytectl/en/stable/gen/flytectl_get_execution.html>`__.
+Flytectl:
+=============
```

Comment:
This should be of the same length as the heading. Please refer to how the above heading looks.